scripts/dependency_checker.py

```
@@ -170,71 +207,72 @@ def default_run():
 
 	# Then build up a dependency graph with a dict: { key: <protobuf file> value: [ included by ] }
-	for latest_file in latest_proto_files:
-		includes = extract_proto_includes(directory_path + latest_file)
+	for proto_file in all_proto_files:
+		if debug:
+			print(f"🔍 Checking the file {proto_file} for includes")
+		includes = extract_proto_includes(directory_path + proto_file)
 		for include in includes:
+			if debug:
+				print(f"  ➡️ Processing include: {include}")
 			if include.startswith("cmp/"): # we're only interested in our includes
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)